helm/templates/log-configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "schema-registry.fullname" . }}-log-configmap
+  labels:
+    {{- include "schema-registry.labels.standard" . | nindent 4 }}
+data:
+  log4j.properties: |-
+    log4j.rootLogger={{ .Values.logging.level }}, stdout
+
+    log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+    log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+    log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
+
+    log4j.logger.kafka=ERROR, stdout
+    log4j.additivity.kafka.server=false
```

Comment:
@kishansairam9 : fixed duplication issues with additivity in pinot log configuration
can you please refer to that and fix this as well and rebase the PR with main branch.

Please note that pinot is using log4j2 while schema registry is using log4j-1.2

Reply:
done

Reply:
I checked the syntax for log4j 1 and wrote this. We can't test this until we merge